packages/xs-extra/wsproxy.master/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "wsproxy"
+version: "1.1.0"
+authors: [ "Jon Ludlam" "jonathan.ludlam@citrix.com"
```

Comment:
Maybe this should be one string, instead of two: `"Jon Ludlam <jonathan.ludlam@citrix.com>"`?